addons/stock/static/src/stock_forecasted/forecasted_details.xml

```diff
@@ -32,20 +32,30 @@
                             class="fw-bold"/>
 
                         <t t-elif="line.reservation">
-                            Reserved from stock
-                            <button t-if="line.move_out and line.move_out.picking_id"
+                            <a t-out="line.reservation.name" href="#" class="fw-bold"
+                               t-on-click.prevent="() => this.props.openView(line.reservation._name, 'form', line.reservation.id)"
+                                /> reserved
+                            <button t-if="displayReserve(line)"
                                 class="btn btn-sm btn-primary o_report_replenish_unreserve"
                                 name="unreserve_link"
-                                t-on-click="() => this._unreserve('stock.picking', line.move_out.picking_id.id)">
+                                t-on-click="() => this._unreserve(line.move_out.id)">
                                 Unreserve
                             </button>
                         </t>
+                        <t t-elif="line.in_transit">
+                            <t t-if="line.move_out">
+                                <span>Stock In Transit</span>
+                            </t>
+                            <t t-else="">
+                                <span>Free Stock in Transit</span>
+                            </t>
+                        </t>
```

Comment:
```suggestion
                        <span t-elif="line.in_transit">
                            <t t-if="line.move_out">Stock In Transit</t>
                            <t t-else="">Free Stock in Transit</t>
                        </span>
```
Also, "*Stock In Transit*" and "*Free Stock in Transit*" should be added in the translation file I think